src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2359,7 +2359,7 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
                 {
                     if (importingModule)
                     {
-                        IList<PSModuleInfo> moduleProxies = ImportModulesUsingWinCompat(new string[] { moduleManifestPath }, null, new ImportModuleOptions());
+                        IList<PSModuleInfo> moduleProxies = ImportModulesUsingWinCompat(new string[] { moduleManifestPath }, null, options);
```

Comment:
Why do we need the change?

Reply:
This is the fix for item 2. This scenario is covered by test `-NoClobber and -Scope work with implicit WinCompat`.